### ISSUE TITLE
Not to use nested string interpolation

### ISSUE THREAD

**Issue by roman-dubrovsky** [rubocop-hq/rubocop#7480]
Hello, 

What do you think about adding a cop which can find examples of using nested interpolation (when we use string interpolation into another string interpolation). Just for me, this is a good example of the bad design and places which should be refactored.

Example,

```ruby
def say_hello(user = nil)
  puts "Hello #{ user.present? ? "dear #{user.first_name}" : "everyone" }. How are you?"
end
```

Thank you!
